### PR TITLE
Update 7.3.5

### PR DESCRIPTION
**Pull request by Arishamays1**
`Jupyter_client` version bump to `7.3.5`. 
- updated sha256
- replaced `setuptools` with `hatchling`, added `wheel`
- updated dependencies

### CODE REVIEW

**Comment by marcoesters** [comment]
You seem to have a merge conflict here

**Comment by Arishamays1** [comment]
> You seem to have a merge conflict here

Thanks Marco, fixed this issue.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -33,21 +33,20 @@ requirements:
     - pyzmq >=23.0
     - tornado >=6.0
```
Comment (marcoesters) [inline recipe/meta.yaml]: `tornado` needs to be at least `6.2`: https://github.com/jupyter/jupyter_client/blob/main/pyproject.toml#L32

**Review by marcoesters — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (Arishamays1) [reply recipe/meta.yaml]: Fixed, thanks!

**Pull request merged**